src/google/cloud/ndb/_datastore_api.py

```diff
@@ -70,6 +71,40 @@ def make_stub(client):
     return datastore_pb2_grpc.DatastoreStub(channel)
 
 
+def make_call(rpc_name, request, retries=None):
+    """Make a call to the Datastore API.
+
+    Args:
+        rpc_name (str): Name of the remote procedure to call on Datastore.
+        request (Any): An appropriate request object for the call, eg,
+            `entity_pb2.LookupRequest` for calling ``Lookup``.
+        retries (int): Number of times to potentially retry the call. If
+            :data:`None` is passed, will use :data:`_retry._DEFAULT_RETRIES`.
+            If :data:`0` is passed, the call is attempted only once.
+
+    Returns:
+        tasklets.Future: Future for the eventual response for the API call.
+    """
+    api = stub()
+    method = getattr(api, rpc_name)
```

Comment:
Why the indirection around method? Is it subject to change out from under us before the tasklet executes?

Reply:
Not sure I understand the question. Why is it done this way as opposed to...?

Reply:
Oh, I see, since rpc_name is dynamic we can't call it directly with api.method syntax. My mistake.